src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -10692,7 +10694,7 @@ void func_8084B78C(Player* this, GlobalContext* globalCtx) {
 
 void func_8084B840(GlobalContext* globalCtx, Player* this, f32 arg2) {
     if (this->actor.wallBgId != BGCHECK_SCENE) {
-        DynaPolyActor* dynaActor = DynaPoly_GetActor(&globalCtx->colCtx, this->actor.wallBgId);
+        DynaPolyActor* dynaActor = (DynaPolyActor*)DynaPoly_GetActor(&globalCtx->colCtx, this->actor.wallBgId);
```

Comment:
It's not a change from this PR, but why does `DynaPoly_GetActor` no longer return a `DynaPolyActor*`? Considering what the function does, I would expect it to return a `DynaPolyActor*`, not an `Actor*`, and it would keep things much easier.

Reply:
yea this is something that will be sorted with bgcheck docs

Reply:
Okay, what's the best thing to do in the meantime, to be consistent? This function is used in 12 files, 7 of which fall under the purview of this PR, so it would be easy enough to change all of them.

Reply:
dont cast and leave the warnings until we sort out the functions return type proper with bgcheck docs